config/install/uiowa_bar.settings.yml

```diff
@@ -1,4 +1,5 @@
 uiowa_bar:
   wordmark: uiowa
+  display_search: true
```

Comment:
I think this will be an integer (0/1) without the `#return_value` set on the checkbox form element.

Reply:
Changed to 1 by default